jdaviz/configs/cubeviz/plugins/slice/tests/test_slice.py

```diff
@@ -114,3 +114,20 @@ def test_indicator_settings(cubeviz_helper, spectrum1d_cube):
 
     sl.show_wavelength = False
     assert indicator.label.visible is False
+
+
+@pytest.mark.filterwarnings('ignore:No observer defined on WCS')
+def test_init_slice(cubeviz_helper, spectrum1d_cube, tmpdir):
```

Comment:
You don't need a temp dir for this test as you are not doing file I/O.

```suggestion
def test_init_slice(cubeviz_helper, spectrum1d_cube):
```

Reply:
good catch, I got sloppy copying directly from the moment maps test.  Fixed.